pennylane/transforms/get_unitary_matrix.py

```diff
@@ -0,0 +1,141 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+A transform to obtain the matrix representation of a quantum circuit.
+"""
+from functools import wraps
+import numpy as np
+from pennylane.wires import Wires
+import pennylane as qml
+from pennylane.math.utils import get_interface
+
+
+def get_unitary_matrix(circuit, wire_order=None):
+    """Construct the matrix representation of a quantum circuit.
```

Comment:
@ingstra the 'anomalous backslash' issue can be solved by prepending the string with an `r` here 🙂 
```suggestion
    r"""Construct the matrix representation of a quantum circuit.
```

This indicates to Python that the string is a 'raw' string, and it will stop attempting to 'interpret' escaped characters.

Reply:
Awesome! Thanks. So many little things to know...